pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample

```diff
@@ -16,6 +16,10 @@
 export SSID='your_ssid'
 export WIFIPASS='your_pass'
 
+# Cellular Connection Quality-of-Life
+# 1. If using a cellular connection, turn on the following flag to have TeslaUSB attempt sync right after each snapshot.
```

Comment:
"1." implies this is part of a list, but it isn't. Also this isn't necessarily related to having a _cellular_ connection, it's _any_ connection.